test/integration/services/alert.js

```diff
@@ -0,0 +1,89 @@
+describe('Alert Service', function() {
+  specify('Displaying', function() {
+    cy
+      .visit('/');
+
+    cy
+      .clock();
+
+    cy
+      .window()
+      .should('have.property', 'Radio')
+      .then(Radio => {
+        Radio.request('alert', 'show:success', 'success');
+      });
+
+    cy
+      .get('.alert-box')
+      .contains('success')
+      .should('exist')
```

Comment:
we actually don't need this line.. `contains` will fail if it's not true.  as long as line 24 can be swapped to `exist` and fail, we're good.